assets/src/edit-story/components/panels/design/captions/captions.js

```diff
@@ -43,6 +45,13 @@ const BoxedTextInput = styled(TextInput)`
   opacity: 1;
 `;
 
+const HighlightRow = styled(Row)`
+  ${({ focusContainerCss }) => focusContainerCss}
```

Comment:
Is there a reason why this doesn't just import constants for `focusContainerCss` and `focusContainerSelector`? It doesn't seem like they change at all.

Reply:
Yeah, I struggled with this and I'm not sure the best approach but I'll go back to this and see how it looks. It seemed weird to always be importing the styles and the hook as a clump rather than getting what you need through the hook, but this is also weird in this way

Reply:
One thing I was trying to accomplish by passing styles/components through the hook is that we can tie the highlighting styles to the highlighted state by providing it through the piece context that we're asking for. If I just import the constants, now I have to do logical checks to determine when to show the styles in the component, rather than offloading it to the context. I'll see if I can get that but in a cleaner way.

Reply:
I can't tell how importing would be any different than what's going on now since these variables don't seem to be used as conditionals or are being overridden. Am I missing something?

Reply:
It's still definitely a problem with the implementation because it's not obvious what's happening here... But the idea is that the existence of state named by keys is what is telling the components when to show the styles and focus the inputs. So, when I select like this:

`const { highlight } = useHighlights(({ storyTitle }) => ({ highlight: storyTitle })`

I'm using the existence of a state called `storyTitle` to determine when to apply the highlighted styles and when to focus inputs. If `storyTitle` doesn't exist as a key in the state, those styles aren't defined in the context and don't get passed on to the component. As a result, the highlighted state only shows when the part of state that I selected when calling `useHighlights` exists. 

For example, if I were to import the styles as constants with the current implementation, every time the story title is focused, whether it's "highlighted" or not, it would show the highlighted styles, because I'm using the `focus-within` selector to define the styles. The styles (as constants) wouldn't be aware of the highlighted context or which part of state is relevant for when the highlighted styles should show.

Reply:
The selector could be a constant, but I think this could be better in general.

Reply:
I will probably end up using constants and passing something like `isHighlighted` props to the components that are being used to determine when to show styles instead as this isn't really clear at all.

Reply:
Definitely still open to comments about how to provide common highlight styles that are extensible. Right now the border radius is the only thing that might look strange, so I allowed that to be overridden via props. Not sure what I have is the right way to provide these styles based on the highlighted state